index.html

```diff
@@ -157,7 +157,7 @@
                                     </a>
                                     {% endif %}
                                 </div>
-                                <p><i class="fa fa-code"></i>{{student.name}}</p>
+                                <p><i class="fa fa-code"></i>&nbsp;{{student.name}}</p>
                             </div>
```

Comment:
Protocol: Do not specify the protocol unless required: Remove the protocol "https:" from the url.

*HTMLLintBear, severity NORMAL, section `html`.*